src/carbonio-files-ui-common/utils/utils.ts

```diff
@@ -101,11 +101,11 @@ export const getIconByFileType = (type: NodeType, subType?: Maybe<string>): stri
 		case NodeType.Root: {
 			switch (subType) {
 				case ROOTS.LOCAL_ROOT:
-					return 'Home';
+					return 'FolderOutline';
```

Comment:
Do not use outline here, this utils is used also to use the filled version of the icons. The outline is defined outside, where the function is called

Reply:
@beawar so basically it just returns 'Folder'?

Reply:
Exactly, because in some place (like the modals) the icons are filled, in other places the icons are outline (like the displayer of the updloads).
If you need to always return the outline version for only a type, it probably makes sense to refactor this function a bit, in order to include an option to get the outline version of the icon, so that each place can decide type by type if the icon should be filled or not. Something like

```ts
export const getIconByFileType = (
	type: NodeType,
	subType?: Maybe<string>,
	options?: { outline?: boolean }
): keyof DefaultTheme['icons'] => {
	function buildIconName(icon: keyof DefaultTheme['icons']): keyof DefaultTheme['icons'] {
		return options?.outline ? `${icon}Outline` : icon;
	}
	switch (type) {
		case NodeType.Folder:
			return buildIconName('Folder');
		case NodeType.Text:
			switch (subType) {
				case 'application/pdf':
					return buildIconName('FilePdf');
				default:
					return buildIconName('FileText');
			}
                 // etc
    }
} 
```